src/views/dashboard/DashboardPage.vue

```diff
@@ -1,9 +1,9 @@
 <template>
```

Comment:
Do you have some modifications in the DashboardPage ?
I can't accept a PR with a file fully edited without any reason, unless in a separate dedicated commit